app/helpers/search_bar_helper.rb

```diff
@@ -0,0 +1,25 @@
+module SearchBarHelper
+  def mount_search_bar(
+    id:,
```

Comment:
should the id actually be a kwarg? i think since its required and usually the only one used you can make it a regular argument (you can mix regular arguments with kwargs, as long as the regular ones are first)

Reply:
Done :)